package.json

```diff
@@ -106,7 +106,7 @@
     "karma-chrome-launcher": "^3.1.1",
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "^5.1.0",
-    "karma-jasmine-html-reporter": "^1.6.0",
+    "karma-jasmine-html-reporter": "^2.0.0",
```

Comment:
This doesn't align with multiple packages in the PR description 🤔

Reply:
Oh what the hell happened 😮